Provider/ImageProvider.php

```diff
@@ -174,6 +174,10 @@ public function generatePublicUrl(MediaInterface $media, $format)
      */
     public function generatePrivateUrl(MediaInterface $media, $format)
     {
+        if ($format == 'reference') {
```

Comment:
I think Yoda it's a Symfony's convention: https://github.com/FriendsOfPHP/PHP-CS-Fixer/commit/4725fec466e9a796186896334d8c87f3bc46f7ee